teamwork/templates/base.html

```diff
@@ -1,23 +1,78 @@
-    <link rel="stylesheet" href="{% static 'css/base.css' %}">
-  </head>
-  <body>
-    {% block content %}
-    {% endblock %}
-  </body>
-</html>
+      <!-- Custom CSS -->
+      <link rel="stylesheet" href="{% static 'css/main.css' %}">
+    </head>
+    <body>
+      <!-- Navigation bar -->
+      <nav class="navbar navbar-inverse navbar-static-top" role="navigation">
+                      <div class="container">
+                          <div class="navbar-header">
+                              <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+                                  <span class="sr-only">Toggle navigation</span>
+                                  <span class="icon-bar"></span>
+                                  <span class="icon-bar"></span>
+                                  <span class="icon-bar"></span>
+                              </button>
+                              <p class="navbar-brand">{{ project.name }}</p>
+                          </div>
+                          <div class="navbar-collapse collapse">
+                              <ul class="nav navbar-nav">
+                                  <li class="active"><a href="{% url 'dashboard' user.username %}">Dashboard</a>
+                                  </li>
+                                  <li><a href="calendar">Calendar</a>
+                                  </li>
+                                  <li><button type="button" class="btn btn btn-default btn-success navbar-btn"> <span class="glyphicon glyphicon-plus"></span> Add new project </button>
+                                  </li>
+                              </ul>
+
+                              <ul class="nav navbar-nav navbar-right">
+                                <li class="dropdown">
+                                    <a href="#" class="dropdown-toggle" data-toggle="dropdown"><span class="glyphicon glyphicon-user"></span> {{ user.username }} <b class="caret"></b></a>
+                                    <ul class="dropdown-menu">
+                                        <li><a href="#"><span class="glyphicon glyphicon-cog"></span> Settings</a>
+                                        </li>
+                                        <li class="divider"></li>
+                                        <li><a href="#">Log out</a>
+                                        </li>
+                                    </ul>
+                                </li>
+                              </ul>
+                          </div>
+                          <!--/.nav-collapse -->
+                      </div>
+        </nav>
+      {% block content %}
+      {% endblock %}
+
+      <!--Footer -->
+      <footer class="container-fluid text-center">
+        <p><a href="https://mycourses.aalto.fi/" title="MyCourse" target="_blank">My Course</a>  |  All rights reserved</p>

```

Comment:
1st comment: I disagree, it's a better that user don't have to press back button to go back to the service or having to open it again. In my opinion, best practice is that all the link to external service should be opening a new tab, not close the service you're using. But if it's compromising security then it's a different story.

3rd comment: All rights reserved doesn't really mean anything in this context, I just put it there so the My Course link doens't look too out-of-the-blue. Other idea is to put 'Created by....' or whatever.